content/en/css.md

```diff
@@ -60,7 +64,7 @@ Do not indent child styles underneath their parent styles for a few reasons. Whe
 
 To simplify CSS authoring, we set the `box-sizing` attribute to `border-box` for all page elements. This enables us to use round numbers for width like 50% and then apply a padding or border to that same element without needing to (1) adjust the width accordingly using calc (since borders use pixels rather than percents) or (2) create an element inside it to take the padding and border. This is the only case where we use the inefficient universal selector (`*`).
 
-```css
+```
```

Comment:
Prism needs "markup" or "css" next to the backticks so the correct CSS class is put on the code block, so the syntax highlighting will be appropriate to the language.